roles/neutron-control/tasks/main.yml

```diff
@@ -27,7 +27,7 @@
   tags: db-migrate
 
 - name: sync neutron database for lbaas
-  command: neutron-db-manage --service lbaas --config-file /etc/neutron/neutron.conf \
+  command: neutron-db-manage --subproject neutron-lbaas --config-file /etc/neutron/neutron.conf \
```

Comment:
Tested RELEASE-3.1.x with package (2016.1-bbc248) in local and met the issue, but don't meet it in master branch:

2017-04-12 09:10:41,854 p=25485 u=ubuntu |  TASK [neutron-control : sync neutron database for lbaas] ***********************
2017-04-12 09:10:41,854 p=25485 u=ubuntu |  Wednesday 12 April 2017  09:10:41 +0000 (0:00:06.934)       0:41:19.461 *******
2017-04-12 09:10:44,331 p=25485 u=ubuntu |  fatal: [controller-0]: FAILED! => {"changed": true, "cmd": ["neutron-db-manage", "--service", "lbaas", "--config-file", "/etc/neutron/neutron.conf", "--config-file", "/etc/neutron/plugins/ml2/ml2_plugin.ini", "upgrade", "head"], "delta": "0:00:01.673791", "end": "2017-04-12 09:10:44.056173", "failed": true, "rc": 2, "start": "2017-04-12 09:10:42.382382", "stderr": "usage: neutron-db-manage [-h] [--config-dir DIR] [--config-file PATH]\n                         [--nosplit_branches] [--split_branches]\n                         [--subproject SUBPROJECT]\n                         [--database-connection DATABASE_CONNECTION]\n                         [--database-engine DATABASE_ENGINE]\n                         {current,history,branches,heads,check_migration,upgrade,downgrade,stamp,revision,has_offline_migrations}\n                         ...\nneutron-db-manage: error: argument command: invalid choice: 'lbaas' (choose from 'current', 'history', 'branches', 'heads', 'check_migration', 'upgrade', 'downgrade', 'stamp', 'revision', 'has_offline_migrations')", "stdout": "", "stdout_lines": [], "warnings": []}